assets/app/scripts/directives/events.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .directive('events', function($routeParams, $filter, DataService, ProjectsService, Logger) {
+    return {
+      restrict: 'E',
+      scope: {
+        resourceKind: "@",
+        resourceName: "@",
+        parentScope: "="
```

Comment:
Don't pass in parent scope. Pass in `projectContext` instead. The directive should know nothing about the parent scope.